### PR TITLE
Add tesseract-ocr to process image messages 

### PR DESCRIPTION
**Pull request by nonumpa**
a. Add tesseract-ocr to process image messages 
1. Extract message_process statements to `processText` function.
2. Save and process image files in tmp_image_process dir.
3. Update README.md about tesseract-ocr configuration. [related project](https://github.com/cofacts/heroku-buildpack-tesseract)

b. Update reply messages for image input 
1. [initState] Don't reply article-found-prefix when `messageType` is not text.
2. [askingArticleSource] Guide user to ask fact-checkers in case `messageType` is not text and is able to send article to database.

---
update : 2019/12/04 20:34
#### tesseract-ocr-4.1.0+tessdata-4.0.0 (my local machine)

"White word" tesseract can't recognize well.

#### tesseract-ocr-4.0.0+tessdata-4.0.0 (Heroku)
But on the Heroku the result is not good as my local machine, maybe I should update Buildpack's tesseract binary version to 4.1.0.

### CODE REVIEW

**Comment by changhc** [comment]
How accurate is it now?

**Comment by nonumpa** [comment]
> How accurate is it now?

update above

File: src/handlers/initState.js [hunk 1]
```diff
@@ -146,9 +150,16 @@ export default async function initState(params) {
       },
       templateMessage,
     ];
+    console.log(JSON.stringify(event));
```
Comment (MrOrz) [inline src/handlers/initState.js]: please remove this console.log, thanks

File: src/handlers/initState.js [hunk 2]
```diff
@@ -133,11 +135,13 @@ export default async function initState(params) {
       },
     };
 
-    replies = [
+    var prefixTextArticleFound = [
```
Comment (LucienLee) [inline src/handlers/initState.js]: `const` would be better

File: src/handlers/initState.js [hunk 3]
```diff
@@ -133,11 +135,13 @@ export default async function initState(params) {
...
       {
         type: 'text',
         text: `🔍 ${t`There are some messages that looks similar to "${articleSummary}" you have sent to me.`}`,
       },
+    ];
+    var textArticleFound = [
```
Comment (LucienLee) [inline src/handlers/initState.js]: `const` would be better

**Pull request merged**